netbox/extras/forms/model_forms.py

```diff
@@ -220,24 +222,44 @@ class Meta:
 
 
 class WebhookForm(NetBoxModelForm):
+
+    fieldsets = (
+        (_('Webhook'), ('name', 'tags',)),
+        (_('HTTP Request'), (
+            'payload_url', 'http_method', 'http_content_type', 'additional_headers', 'body_template', 'secret',
+        )),
+        (_('SSL'), ('ssl_verification', 'ca_file_path')),
+    )
+
+    class Meta:
+        model = Webhook
+        fields = '__all__'
+        widgets = {
+            'additional_headers': forms.Textarea(attrs={'class': 'font-monospace'}),
+            'body_template': forms.Textarea(attrs={'class': 'font-monospace'}),
+        }
+
+
+class EventRuleForm(NetBoxModelForm):
     content_types = ContentTypeMultipleChoiceField(
         label=_('Content types'),
         queryset=ContentType.objects.all(),
         limit_choices_to=FeatureQuery('webhooks')
```

Comment:
This will need to be updated for #14153 and to reference "event rules" as a feature.